torch/_dynamo/variables/misc.py

```diff
@@ -814,10 +919,25 @@ def call_method(
             return variables.ConstantVariable.create(None)
 
         if name != "save_for_backward":
-            unimplemented(f"autograd.Function context method: {name}")
+            unimplemented_v2(
+                gb_type="Unsupported autograd.Function context method",
+                context=f"call_method {self} {name}",
+                explanation="Dynamo does not support calling the method "
+                f"`{name}` on `autograd.Function` context objects. Supported "
+                "methods are `__setattr__`, `save_for_backward` and "
+                "`mark_non_differentiable`.",
+                hints=[*graph_break_hints.USER_ERROR],
+            )
         if self.saved_tensors is None:
-            unimplemented(
-                "save_for_backward only supported on a newly constructed FunctionCtx"
+            unimplemented_v2(
+                gb_type="Unsupported autograd.Function context usage",
+                context=f"call_method {self} {name}",
+                explanation="Dynamo requires the `saved_tensors` attribute "
+                "to be initialized on the `autograd.Function` context object.",
+                hints=[
+                    "Ensure that the `saved_tensors` attribute is properly "
+                    "initialized before calling `save_for_backward`.",
```

Comment:
Also mention the original graph break message "save_for_backward only supported on a newly constructed autograd.Function context" somewhere